src/app/contract/contractParser.js

```diff
@@ -42,7 +43,7 @@ var getDetails = function (contractName, contract, source) {
   }
 
   if (source && contract.assembly !== null) {
-    detail['Assembly'] = formatAssemblyText(contract.evm.legacyAssembly, '', source.content)
+    detail['Assembly'] = solcTranslate.prettyPrintLegacyAssemblyJSON(contract.evm.legacyAssembly, '', source.content)
```

Comment:
does `prettyPrintLegacyAssemblyJSON` could also display the corresponding source code ?

Reply:
I just tried and couldn't see the associated source code ...

Reply:
Sorry I'm not sure I get it ?

Reply:
Sorry, it was a typo, looked at the wrong function:https://github.com/ethereum/solc-js/blob/master/translate.js#L187

My change should be:
```
     detail['Assembly'] = solcTranslate.prettyPrintLegacyAssemblyJSON(contract.evm.legacyAssembly, source.content) 
```

Then it will work.

Reply:
Fixed, again! 😉